types/sinon/index.d.ts

```diff
@@ -79,6 +79,7 @@ declare namespace Sinon {
         alwaysReturned(obj: any): boolean;
         invokeCallback(...args: any[]): void;
         getCall(n: number): SinonSpyCall;
+        getCalls(): SinonSpyCall[];
```

Comment:
In a well-documented package you shouldn't include something that's not documented and may change.